lib/App/Sqitch/Engine/Upgrade/clickhouse-1.0.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE releases (
+    version         REAL COMMENT 'Version of the Sqitch registry.' PRIMARY KEY,
+    installed_at    DATETIME64(6, 'UTC') NOT NULL DEFAULT now64(6, 'UTC')
+                    COMMENT 'Date the registry release was installed.',
+    installer_name  TEXT                 NOT NULL
+                    COMMENT 'Name of the user who installed the registry release.',
+    installer_email TEXT                 NOT NULL
+                    COMMENT 'Email address of the user who installed the registry release.'
+) ENGINE = MergeTree
+  SETTINGS enable_block_number_column = 1, enable_block_offset_column = 1
```

Comment:
fine to only define primary key without ORDER BY?

[docs](https://clickhouse.com/docs/engines/table-engines/mergetree-family/mergetree) make it seem like ORDER BY is required, outside of using a setting which will `ORDER BY tuple()` with suboptimal effects

Reply:
Yeah, my first day of work I was in a training session where they used `PRIMARY KEY` in most of the examples. Seems to be more or less the same thing as `ORDER BY`. Or at least [it is in the absence of `ORDER BY`](https://clickhouse.com/docs/best-practices/choosing-a-primary-key).

Reply:
Primary Key is sparse index, `ORDER BY` is physical sorting of data. Very confusing I know

Reply:
Queries most often order by date, but deletions are done by primary key. What _should_ it be?

Reply:
I think you want both